index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+    
+    <header class="title">
+        <ul>
+            <li>
+                <h1 class="main-text">Weather Report for  
+                </h1> </li>
+                <li> <h4 id="locationDisplay">
+                    Unceded Coast Salish Land</h4> </li>
+                
+                    <li> <h2 id= "sassy-subtitle"> *Whether or not this report matches the weather you are experiencing is completely coincidental* </h2> </li>
+            </ul>
+            </div>
+        </figure>
+    </header>
+
+    <section class="weather-display">
+        <div class="left-column">
+
+        <div class="weather-item" id=temperature>
+            <h3 id="temperatureCount">Temperature 65℉</h3>
```

Comment:
Consider adding an initial color so that when the page is loaded the color matches the 65℉ temperature